lasagne/objectives.py

```diff
@@ -1,35 +1,50 @@
+import theano
 import theano.tensor as T
 
 
-def mse(x, t):
+def mse(x, t, m):
     """Calculates the MSE mean across all dimensions, i.e. feature
      dimension AND minibatch dimension.
 
     :parameters:
         - x : predicted values
         - t : target values
+        - m : mask; None for no mask, or a 1-dimensional array, that
+            selects which rows in `x` and `t` should count towards the
+            result
```

Comment:
Does it have to be a 1-dimensional array? I'd prefer being able to use it with a mask the same shape as `t`.